src/Kiota.Builder/Writers/Php/CodeClassDeclarationWriter.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Linq;
+using Kiota.Builder.Extensions;
+
+namespace Kiota.Builder.Writers.Php
+{
+    public class CodeClassDeclarationWriter: BaseElementWriter<CodeClass.Declaration, PhpConventionService>
+    {
+
+        public CodeClassDeclarationWriter(PhpConventionService conventionService) : base(conventionService) { }
+
+        public override void WriteCodeElement(CodeClass.Declaration codeElement, LanguageWriter writer)
+        {
+            PhpConventionService.WritePhpDocumentStart(writer);
```

Comment:
I know sonarqube says to switch to static method but this is the one rule I don't like from methods that are called from outside of the class. Makes mocking harder should we ever need too. Sorry for not being specific enough in my original comment. Would you mind reverting that set of changes?